web-ui/src/main/resources/catalog/components/thesaurus/partials/keywordselector.html

```diff
@@ -14,7 +14,7 @@
 
   <span data-ng-show="!invalidKeywordMatch">
     <div class="row">
-      <label class="col-sm-2 control-label">
+      <label class="col-sm-2 control-label" data-gn-field-tooltip="{{schema}}|{{thesaurusKey}}">
```

Comment:
Should we use the description in the thesaurus?

![image](https://user-images.githubusercontent.com/1701393/162962258-3e0ba825-af1d-4d57-a3fa-491d7d47056a.png)

For INSPIRE registry, import is preserving description.